From e7528a65cd4279a102755e556d2763d01f0d7b92 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:16:56 +0800
Subject: [PATCH] Revert "USB: gadget: android: check for null _android_dev in
 android_register_function()"

This reverts commit 4bcc593ad967fd7ff1c30f61cb6ea5dc91b8c1a0.
---
 drivers/usb/gadget/android.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/android.c b/drivers/usb/gadget/android.c
index 3c4d44d663c0..70e48ac3e2a9 100755
--- a/drivers/usb/gadget/android.c
+++ b/drivers/usb/gadget/android.c
@@ -313,7 +313,7 @@ void android_register_function(struct android_usb_function *f)
 	/* bind our functions if they have all registered
 	 * and the main driver has bound.
 	 */
-	if (dev && dev->config && _registered_function_count == dev->num_functions)
+	if (dev->config && _registered_function_count == dev->num_functions)
 		bind_functions(dev);
 }
 
-- 
2.35.3

